src/PowerShell/Microsoft.WinGet.Client.Engine/Common/WinGetIntegrity.cs

```diff
@@ -46,16 +46,19 @@ public static void AssertWinGet(PSCmdlet psCmdlet, string expectedVersion)
                 var result = wingetCliWrapper.RunCommand("--version");
                 result.VerifyExitCode();
             }
-            catch (Win32Exception)
+            catch (Win32Exception e)
             {
+                psCmdlet.WriteDebug($"'winget.exe --version' Win32Exception {e}");

```

Comment:
NIT: I think it would be better to just say `Failed to call winget.exe with Win32Exception {e}`. I don't think the --version is important and might be confusing.

Reply:
Same on line 56 and 61